### PR TITLE
[enterprise-4.14] CNV-26426: Release note: Deprecate TTO

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/CNV-26426

Link to docs preview:
https://64962--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-deprecated

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -130,6 +130,7 @@ The SVVP Certification applies to:
 Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 //CNV-26426 [DOCS] Release note: Deprecate TTO
+* The `tekton-tasks-operator` is deprecated and tekton tasks are now deployed by the `ssp-operator`.
```
Comment (geetikakay) [inline virt/release_notes/virt-4-14-release-notes.adoc:133]: maybe replace tekton tasks with tekton tasks and example pipelines

Comment (geetikakay) [reply virt/release_notes/virt-4-14-release-notes.adoc:133]: @ksimon1 wdyt^^

Comment (ksimon1) [reply virt/release_notes/virt-4-14-release-notes.adoc:133]: I agree with suggestion Geetika proposed

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 2]
```diff
@@ -140,6 +140,7 @@ The SVVP Certification applies to:
 Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 //CNV-26426 [DOCS] Release note: Deprecate TTO
+* The `tekton-tasks-operator` is deprecated and tekton tasks and example pipelines are now deployed by the `ssp-operator`.
```
Comment (mburke5678) [inline virt/release_notes/virt-4-14-release-notes.adoc:143]:
```suggestion
* The `tekton-tasks-operator` is deprecated and Tekton tasks and example pipelines are now deployed by the `ssp-operator`.
```

**Comment by mburke5678** [comment]
@danielclowers One possible nit. Otherwise LGTM

**Comment by geetikakay** [comment]
lgtm

**Pull request merged**